src/libs/actions/Task.js

```diff
@@ -359,8 +357,9 @@ function reopenTask(taskReportID, taskTitle) {
  * @param {String} editedTask.description
  * @param {String} editedTask.assignee
  * @param {Number} editedTask.assigneeAccountID
+ * @param {Object} assigneeChatReport - The chat report between you and the assignee
  */
```

Comment:
Check the other params `editedTask` is not needed here

Reply:
bump on this

Reply:
Removed